### ISSUE TITLE
'dotnet' is not recognized as an internal or external command

### ISSUE THREAD

**Issue by ngocan0905** [dotnet/core#6683]
Problem encountered on https://dotnet.microsoft.com/learn/dotnet/hello-world-tutorial/install
Operating System: windows

Provide details about the problem you're experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

**Comment by ngocan0905** [comment]
i've a problem " 'dotnet' is not recognized as an internal or external command,
operable program or batch file."

**Comment by mairaw** [comment]
Did you open a new command prompt after installing the .NET SDK?

**Comment by GGAlanSmithee** [comment]
TLDR: The dotnet.exe executable is missing when installing the dotnet SDK using the official windows installer

I am experiencing the same issue. To add to this conversation here are some things that I have tried and noticed:

1. I've installed the SDK using the official downloadlink [here](https://dotnet.microsoft.com/download/dotnet/5.0)
2. After downloading and installing the SDK, I made sure the system path was setup correctly (see image one)
3. According to [this](https://stackoverflow.com/questions/63746838/net-sdks-not-installing-correctly) and other SO posts, I made sure the path to the x64 SDK was listed before the x86 one (again, see image one)
4. Exploring the dotnet installation folder, the dotnet.exe executable is missing (see image two)
5. According to the original answer in the above SO post, i downloaded and copied the binaries to the correct place, which did include the executable, and that made it work (see image three and image five)

So, in conclusion, it seems like the dotnet.exe file is missing when installing the SDK via the official installer. Not sure if this is related to my OS (see image 4) but this has worked without issues before.

The one difference that I can think of is that I uninstalled all versions of Visual Studio (except VS code) prior to re-installing these SDK:s. (But even so, I have been using the dotnet cli with VS Code for a long time before this)

![image](https://user-images.githubusercontent.com/11805103/133378886-ceb303d5-c77c-47ee-99ca-eb5a3c04983d.png)

![image](https://user-images.githubusercontent.com/11805103/133378571-094c4ee7-79af-4ee0-9a3a-8de689a0ba6d.png)

![image](https://user-images.githubusercontent.com/11805103/133379286-623b4424-129d-40c9-bee6-8f2849ca9a44.png)

![image](https://user-images.githubusercontent.com/11805103/133379563-b9555eb0-1d92-4e05-9b28-393628e88ecf.png)

![image](https://user-images.githubusercontent.com/11805103/133379771-ceab9423-93f4-460e-ac7f-0e09485b1e76.png)

**Comment by GGAlanSmithee** [comment]
@mairaw this issue was auto closed, but please see my response, or should I open a new issue instead?

**Comment by mairaw** [comment]
The issue was auto closed because the issue opener didn't respond. It's better to open a new issue since your case might be different than theirs.

**Comment by RussKie** [comment]
I just experienced the same issue on a VM attempting to install [6.0.200](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-6.0.200-windows-x64-installer) - `dotnet.exe` is missing
![image](https://user-images.githubusercontent.com/4403806/155058845-85d52f21-271a-4a92-9c1e-edfc05399ded.png)
